src/Datadog.Trace/Tags.cs

```diff
@@ -10,7 +10,7 @@ public static class Tags
         /// <summary>
         /// The environment of the profiled service.
         /// </summary>
-        public const string Env = "env";
+        public const string Env = CoreTags.Env;
 
         /// <summary>
         /// The version of the profiled service.
```

Comment:
Can you also add this `"version"` tag to the CoreTags? Now that we're targeting the service/env/version trio